framework/src/org/checkerframework/framework/type/AnnotatedTypeLoader.java

```diff
@@ -0,0 +1,324 @@
+package org.checkerframework.framework.type;
+
+import java.io.File;
```

Comment:
Use the Eclipse import ordering from checker-framework/checker/eclipse/checker-framework.importorder